client/components/ui/topbar/QuotaPreviewer.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+export interface QuotaPreviewerProps {
+  quota: number
+  max: number
+  unlimited?: boolean
+}
+
+const { quota, max, unlimited = computed(() => false) } = definePropsRefs<QuotaPreviewerProps>()
+const { locale, t } = useI18n()
+
+const quotaFormat = computed(() => humanSize(quota.value, locale.value, t))
+const maxFormat = computed(() => humanSize(max.value, locale.value, t))
+</script>
+
+<template>
+  <div class="flex flex-col justify-center items-center gap-1.5 h-full quota">
+    <PProgressBar style="height: .5em; width: 100%" :value="unlimited ? 100 : quota / max * 100" :show-value="false" />
+    <span v-if="!unlimited" class="text-xs" style="margin-bottom: -.25rem;">{{ quotaFormat }} / {{ maxFormat }}</span>
+    <span v-else class="text-xs" style="margin-bottom: -.25rem;">{{ t('unlimited-quota') }}</span>
```

Comment:
```suggestion
    <span v-if="!unlimited" class="text-xs mb--1">{{ quotaFormat }} / {{ maxFormat }}</span>
    <span v-else class="text-xs mb--1">{{ t('unlimited-quota') }}</span>
```

same, 'style' attr is doodoocaca